### PR TITLE
hyperzoo dockerfile add cluster-serving

### PR DESCRIPTION
**Pull request by pinggao187**

### CODE REVIEW

File: docker/hyperzoo/Dockerfile [hunk 1]
```diff
@@ -32,7 +34,8 @@ ENV HADOOP_CONF_DIR                     /opt/hadoop-conf
 ENV ANALYTICS_ZOO_HOME                  /opt/analytics-zoo-${ANALYTICS_ZOO_VERSION}
 ENV JAVA_HOME                           /opt/jdk
 ENV REDIS_HOME                          /opt/redis-5.0.5
-ENV PYTHONPATH                          ${ANALYTICS_ZOO_HOME}/lib/analytics-zoo-bigdl_${BIGDL_VERSION}-spark_${SPARK_VERSION}-${ANALYTICS_ZOO_VERSION}-python-api.zip:/opt/models/research/slim
+ENV CS_HOME     /opt/work/cluster-serving
```
Comment (glorysdj) [inline docker/hyperzoo/Dockerfile:37]: format this line?

File: docker/hyperzoo/Dockerfile [hunk 2]
```diff
@@ -45,11 +48,11 @@ RUN apt-get update --fix-missing && \
     chgrp root /etc/passwd && chmod ug+rw /etc/passwd
 
 #java
-RUN wget http://enos.itcollege.ee/~jpoial/allalaadimised/jdk8/jdk-8u241-linux-x64.tar.gz && \
-    gunzip jdk-8u241-linux-x64.tar.gz && \
-    tar -xf jdk-8u241-linux-x64.tar -C /opt && \
-    rm jdk-8u241-linux-x64.tar && \
-    ln -s /opt/jdk1.8.0_241 /opt/jdk
+RUN wget https://enos.itcollege.ee/~jpoial/allalaadimised/jdk8/jdk-8u261-linux-x64.tar.gz  && \
```
Comment (glorysdj) [inline docker/hyperzoo/Dockerfile:51]: 261 is good.

File: docker/hyperzoo/Dockerfile [hunk 3]
```diff
@@ -114,19 +117,46 @@ RUN /opt/download-analytics-zoo.sh && \
...
-    mkdir -p /opt/cluster-serving/test_image && \
-    unzip -p $ANALYTICS_ZOO_HOME/lib/*.zip zoo/serving/quick_start.py > /opt/cluster-serving/quick_start.py
-ADD https://raw.githubusercontent.com/intel-analytics/analytics-zoo/master/pyzoo/test/zoo/resources/serving_quick_start/cat1.jpeg /opt/cluster-serving/test_image 
-ADD https://raw.githubusercontent.com/intel-analytics/analytics-zoo/master/pyzoo/test/zoo/resources/serving_quick_start/dog1.jpeg /opt/cluster-serving/test_image
-ADD https://raw.githubusercontent.com/intel-analytics/analytics-zoo/master/pyzoo/test/zoo/resources/serving_quick_start/fish1.jpeg /opt/cluster-serving/test_image 
-ADD https://sourceforge.net/projects/analytics-zoo/files/analytics-zoo-models/tf-res50/frozen_inference_graph.pb /opt/cluster-serving/model/tf_res50/frozen_inference_graph.pb
-ADD https://sourceforge.net/projects/analytics-zoo/files/analytics-zoo-models/tf-res50/graph_meta.json /opt/cluster-serving/model/tf_res50/graph_meta.json
-RUN chmod a+x ${ANALYTICS_ZOO_HOME}/bin/cluster-serving/*
-RUN cd /opt/cluster-serving && cluster-serving-init && cd ..
+#RUN mkdir -p /opt/cluster-serving && \
```
Comment (glorysdj) [inline docker/hyperzoo/Dockerfile:120]: why keep the ## code?

**Comment by glorysdj** [comment]
need to test on K8S

**Comment by pinggao187** [comment]
jenkins pr test link:
http://10.239.47.210:18888/view/ZOO-PR/job/ZOO-PR-Deploy-Docker/12/console

**Comment by Le-Zheng** [comment]
hyperzoo image is available on k8s test.

File: docker/hyperzoo/Dockerfile [hunk 4]
```diff
@@ -114,19 +117,32 @@ RUN /opt/download-analytics-zoo.sh && \
...
+
+ADD resources/ /opt/work/cluster-serving/resources/
+ADD *.py /opt/work/cluster-serving/
+ADD perf/ /opt/work/perf/
+ADD freeze_checkpoint.py /opt/work
+
+RUN chmod a+x /opt/work/cluster-serving/*
+
+
+ENV OMP_NUM_THREADS     all
```
Comment (qiuxin2012) [inline docker/hyperzoo/Dockerfile:141]: Delete this, `OMP_NUM_THREADS` is set in line 15.

**Pull request merged**